lib/mayaUsd/commands/editTargetCommand.h

```diff
@@ -29,17 +29,29 @@ namespace Impl {
 class SetEditTarget;
 }
 
-class MAYAUSD_CORE_PUBLIC EditTargetCommand : public MPxCommand {
+class EditTargetCommand : public MPxCommand {
 public:
     // plugin registration requirements
+    MAYAUSD_CORE_PUBLIC
```

Comment:
It seems heavy-handed and doesn't examplain why no issue why the LayerEditorCommand, which is written almost the same way.
I suggest you instead try to change the std::unique_ptr<Impl::SetEditTarget> to std::shared_ptr instead. It's probably an issue with the definition of unique_ptr in microsoft